projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/interactions/cartesian-explorer-selection-handler.model.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  PopoverBackdrop,
+  PopoverFixedPositionLocation,
+  PopoverPositionType,
+  PopoverRef,
+  PopoverService
+} from '@hypertrace/components';
+import { BOOLEAN_PROPERTY, Model, ModelProperty } from '@hypertrace/hyperdash';
+import { ModelInject } from '@hypertrace/hyperdash-angular';
+import { Observable, of } from 'rxjs';
+import { CartesianSelectedData } from '../../../../../../public-api';
```

Comment:
bad import. Please go through all changes and verify that no imports are either from any `public-api` file, importing the project they reside in `@hypertrace/observability`, in this case, or importing relative paths across projects.